ui/app/AppLayouts/Onboarding/stores/OnboardingStore.qml

```diff
@@ -14,6 +14,11 @@ QtObject {
         onBoardingModul.setSelectedAccountByIndex(selectedAccountIdx)
     }
 
+    function getPasswordStrengthScore(password) {
+        let userName = onBoardingModul.importedAccountAlias
```

Comment:
I know this isn't your fault, but this property name triggers me with its multiple typos.
Maybe we can rename `onBoardingModul` to `onboardingModuleInst` like we do in the other stores?

Reply:
Yeah! I agree! As we are working in the same files with @alexandraB99  regarding the Onboarding flow.. I prefer to keep it here as it is and we can manage it in her's [PR](https://github.com/status-im/status-desktop/pull/4985) for not creating so much conflicts.